shows_app/src/app/components/feature/reviews/ReviewItem/ReviewItem.tsx

```diff
@@ -10,16 +10,20 @@ import {
 } from "@chakra-ui/react";
 import { StarIcon } from "@chakra-ui/icons";
 import { IReview } from "../../../../typings/reviews";
+import useSWRMutation from "swr/mutation";
+import { swrKeys } from "@/fetchers/swrKeys";
+import { mutator } from "@/fetchers/mutators";
 
 export interface IReviewItemProps {
   review: IReview;
-  onDelete: (review: IReview) => void;
+  onDelete: (reviewId: string, userId: string) => void;
 }
 
 export const ReviewItem = ({ review, onDelete }: IReviewItemProps) => {
   const handleDelete = () => {
-    onDelete(review);
+    onDelete(review.id, review.user.id);
```

Comment:
We are now using SWR to add, edit, and delete reviews. 
We no longer need to maintain reviews state on our own because the true state is kept on the server. Additionally, SWR keeps the data in the cache so that is practically our state.

That is why we can now simplify a bit and move the useSwrMutation that deletes review items in this component. No need to pass the callbacks around anymore.